components/rating/rating.css

```diff
@@ -0,0 +1,23 @@
+/* @import url('../../components/root.css'); */
```

Comment:
Remove this comment if you are not planning to use it.

Reply:
> This is looking nice. Add the javascript snippet as well.

It's a Pure CSS implementation.